README.md

```diff
@@ -502,6 +511,115 @@ server:start()
 
 [socket_ref]: https://www.tarantool.io/en/doc/latest/reference/reference_lua/socket/#socket-tcp-server
 
+## Roles
+
+New roles could be accessed from this project.
+
+### `roles.httpd`
+
+This role can be used only with Tarantool 3. It allows configuring one or more
+HTTP servers. Those servers could be reused by several other roles.
+
+Example of the configuration:
+
+```yaml
+roles_cfg:
+  roles.httpd:
+    default:
+    - listen: 8081
+    additional:
+    - listen: '127.0.0.1:8082'
+```
+
+Server address should be provided either as a URI or as a single port
+(in this case, `0.0.0.0` address is used).
+
+User can access every working HTTP server from the configuration by name,
+using `require('roles.httpd').get_server(name)` method.
+If the `name` argument is `nil`, the default server is returned
+(its name should be equal to constant
+`require('roles.httpd').DEFAULT_SERVER_NAME`, which is `"default"`).
+
+Let's look at the example of using this role. Consider a new role
+`roles/hello_world.lua`:
+```lua
+local M = {}
+local server = {}
+
+M.validate = function()  end
+
+M.apply = function(conf)
+    server = require('roles.httpd').get_server(conf.httpd).httpd
+
+    server:route({
+        path = '/hello/world',
+        name = 'greeting',
+    }, function(tx)
+        return tx:render({text = 'Hello, world!'})
+    end)
+end
+
+M.stop = function()
+    server:delete('greeting')
+end
+
+return M
```

Comment:
```suggestion
local M = {}

M.validate = function(conf)
    if conf == nil or conf.httpd == nil then
        error("httpd must be set")
    end
    local server = require('roles.httpd').get_server(conf.httpd).httpd
    if server == nil then
        error("the httpd server " .. conf.httpd .. " not found")
    end
end

M.apply = function(conf)
    local server = require('roles.httpd').get_server(conf.httpd).httpd

    server:route({
        path = '/hello/world',
        name = 'greeting',
    }, function(tx)
        return tx:render({text = 'Hello, world!'})
    end)
end

M.stop = function()
    local server = require('roles.httpd').get_server(conf.httpd).httpd    
    server:delete('greeting')
end

return M
```

Reply:
It would be nice to add a dependency `roles.httpd` for the role too:
https://www.tarantool.io/en/doc/latest/platform/app/app_roles/#role-dependencies